modules/server-endpoints/language-server/src/main/java/org/wso2/carbon/identity/developer/lsp/debug/runtime/translators/DefaultVariableTranslator.java

```diff
@@ -56,8 +68,16 @@ public Object translate(Object object, int variablesReference) {
             responsedetails.put("body", this.getResponseBody(httpServletResponse));
             responsedetails.put("variablesReference", variablesReference);
             return responsedetails;
+        } else if (object instanceof String) {
+            if (object != null) {
+                try {
+                     return  (Object) new String(Base64.getDecoder().decode((String) object));
+                } catch (IllegalArgumentException e) {
+                    return object;
```

Comment:
We need to add a debut log at least to get an idea why the illigal argument exception occurs